src/modules/sub-items/sub.items.module.js

```diff
@@ -557,6 +718,23 @@ export default class SubItemsModule extends Component {
         );
     }
 
+    onPageChange(page) {
+        this.updateListViewHeight();
+        this.setState(() => ({
+            activePage: page - 1,
```

Comment:
Why activePage is page - 1?

Reply:
Because for pagination the page numbers starts from 1 and is passing `n`, not `n-1`. I was thinking about it and your comment convinced me that it should be changed.